frontend/src/components/module-viewer/content/blocks/column/ColumnBlock.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Grid } from "@chakra-ui/react";
+
+import {
+  ContentBlockProps,
+  ColumnBlockState,
+} from "../../../../../types/ContentBlockTypes";
+import ColumnContent from "./ColumnContent";
+
+const ColumnBlock = ({
+  block: { content, id },
+  index,
+  editable,
+}: ContentBlockProps<ColumnBlockState>): React.ReactElement => {
+  if (index === undefined) {
+    throw Error("Index passed to column component was undefined");
+  }
+
+  // If we are on the learner's side and one of the columns is empty
+  if (!editable && (content.left === null || content.right === null)) {
+    // Figure out which column has content, if any
+    const side = content.left ? "left" : "right";
+    // If both columns are empty, don't render anything
+    if (!content[side]) return <></>;
```

Comment:
Does this look okay? I'm wondering if there might be leftover padding.

Reply:
I'd like to leave this logic in the ColumnBlock if possible. Maybe we could remove the padding from the ContentBlock? Idk, if not I can just move this logic to ContentBlock

Reply:
Is there an actual padding issue? You might be right that we should just remove the padding if that's the case.